### PR TITLE
SMS Gateway

### PR DESCRIPTION
**Pull request by SkylerSeamans**

### CODE REVIEW

File: TeamSnapSDK/SDK/DataTypes/TSDKSmsGateway.m [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//
+
+#import "TSDKSmsGateway.h"
+
+@implementation TSDKSmsGateway
+
+@dynamic domain, name;
+
++ (NSString *)SDKType {
+    return @"SmsGateway";
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataTypes/TSDKSmsGateway.m]: should probably be "sms_gateway"

File: TeamSnapSDK/SDK/DataTypes/TSDKSmsGateway.m [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
++ (NSString *)SDKREL {
+    return @"sms_gateways";
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataTypes/TSDKSmsGateway.m]: one SDKType is sms_gateway SDKRel isn't needed. it's only needed when the plural isn't formed by adding an S

Comment (SkylerSeamans) [inline TeamSnapSDK/SDK/DataTypes/TSDKSmsGateway.m]: these types are correct. SMSGateway is an outlier with the type of SmsGateway and rel of sms_gateways.

Comment (jrahaim) [inline TeamSnapSDK/SDK/DataTypes/TSDKSmsGateway.m]: UGH.. and UGH..

**Comment by jrahaim** [comment]
TSDKSmsGateway.h needs to be a public header.

**Pull request merged**